langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -313,11 +444,11 @@ public isolated function padZero(string str, int len, Char zeroChar = "0") retur
 
 # True if there is a match of `re` against all of `str`.
 # Use `includesMatch` to test whether `re` matches somewhere in `str`.
-public function matches(string str, RegExp 're) returns boolean {
+public isolated function matches(string str, RegExp 're) returns boolean {
    return 're.isFullMatch(str);
 }
 
 # True if there is a match for `re` anywhere in `str`
-public function includesMatch(string str, RegExp 're, int startIndex = 0) returns boolean {
+public isolated function includesMatch(string str, RegExp 're, int startIndex = 0) returns boolean {
    return 're.find(str, startIndex) != ();
 }
```

Comment:
If we are not adding examples now, should we create an issue to add examples for the regex langlib and mention these there?

Reply:
Sure, I'll check on these.

Reply:
Created an issue, https://github.com/ballerina-platform/ballerina-lang/issues/38969 .